compiler/natives/fs_vfsdata.go

```diff
@@ -21,583 +21,635 @@ var FS = func() http.FileSystem {
 		},
 		"/src/net/http/http.go": &vfsgen۰CompressedFileInf
```

Comment:
This file shouldn't be added.

Reply:
GitHub seems to be displaying this comment at an incorrect spot in the discussion tab (it's not the same as in the "Files changed" tab). I left it on line 617, but it's showing it on line 346 here. 😕

To be clear, the two files I was referring to are:

- `/src/testing/testing_test.go~`
- `/src/internal/fmtsort/fmtsort_test.go~`

Reply:
Nice catch: done.

Can we fix the generator to skip such files?

Reply:
Yes, it's Go code, and it's possible to implement any logic for what files to include/exclude. See here:

https://github.com/gopherjs/gopherjs/blob/847fc94819f9d5a4e7154e2203b36f6dbace6f48/compiler/natives/fs.go#L15-L21

I'm not sure if it's worth it, but I'll let you decide.